webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
I think the problem it's not related to the plugin, the error is just related to you don't install the required packages(in this case it's file loader and the style loader) Try `npm install` then build the project before reviewing. Because this isn't a production build, and you can see on your error pictures, it's trying to reach /users/burhanturker :D I'm totally sure about your machine cannot find it :D

Reply:
![Screen Shot 2019-06-18 at 5 06 33 PM](https://user-images.githubusercontent.com/46192266/59691727-26285e80-91ec-11e9-877e-6c0d14324d12.png)

Reply:
Hi @btuerker. It's one of the common sense things I do when working with projects that show dependencies in them in the `package.json` file -> `yarn` or `npm install`. So yeah I did that previously and the error I showed before was post running `yarn` (which is basically the same as `npm install`. 

And yeah still the same question I had, if we aren't using `HtmlWebpackPlugin`, then why did we have it here in the `webpack.config.js`? I was assuming that you had it there because you intended to use it.

At any rate, I deleted the `dist` folder that I had previously locally and re-ran webpack and it now worked.